src/sensitivities/functional/reducedim.jl

```diff
@@ -65,3 +66,41 @@ import Base: mapreduce, sum
         return 2ȳ .* A
     end
 end
+
+@explicit_intercepts(
+    mean,
+    Tuple{Function, AbstractArray{<:∇Scalar}},
+    [false, true],
+    #(dims=:,)  # https://github.com/JuliaLang/julia/issues/31412
+)
+@explicit_intercepts(
+    mean,
+    Tuple{AbstractArray{<:∇Scalar}},
+    [true],
+    (dims=:,)
+)
+
+_denom(x, dims::Colon) = length(x)
+_denom(x, dims::Integer) = size(x, dims)
```

Comment:
Hm, the coverage report says that this and the line below are not covered, which is weird. I deliberately used tests that I thought would exercise these methods. I'll have to take a closer look at this.

Reply:
Are they inlined maybe?

Reply:
Hm, maybe. I'll look more at this tomorrow.

Reply:
The coverage report indeed does not lie; these lines are not covered. Will fix.